gordo_components/builder/build_model.py

```diff
@@ -240,12 +241,23 @@ def _build(self) -> Tuple[Optional[sklearn.base.BaseEstimator], dict]:
                 scaler = self.evaluation_config.get("scoring_scaler")
                 metrics_dict = self.build_metrics_dict(metrics_list, y, scaler=scaler)
 
+                # Generate metadata about CV train, test splits
+                tssplit = TimeSeriesSplit(n_splits=3)
+                for i, (train_ind, test_ind) in enumerate(tssplit.split(X)):
+                    prefix = f"fold-{i+1}"
+                    splits.update(
+                        {
+                            f"{prefix}_train_start": X.index[train_ind[0]].isoformat(),
```

Comment:
I don't know if the indexes are _always_ going to be datetime instances, we definitely don't assume that in other places